nimble/controller/src/ble_ll_dtm.c

```diff
@@ -23,12 +23,24 @@
 
 #include "assert.h"
 #include "os/os.h"
+#include "stats/stats.h"
 #include "controller/ble_ll.h"
 #include "controller/ble_phy.h"
 #include "controller/ble_ll_sched.h"
 #include "controller/ble_ll_xcvr.h"
 #include "ble_ll_dtm_priv.h"
 
+STATS_SECT_START(ble_ll_dtm_stats)
```

Comment:
These stats are not getting registered.

Reply:
right, I usually view stats directly on debugger so forgot about this. will fix :)